src/chi/components/tabs/tabs.scss

```diff
@@ -201,6 +201,98 @@
       }
     }
   }
+
+  &.-list {
+    > li {
+      .chi-tabs_item {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: column;
+
+        &-title {
+          font-weight: $font-weight-semibold;
+        }
+
+        &-text {
+          font-weight: $font-weight-normal;
+        }
+      }
+    }
+  }
+
+  &.-portal {
+    > li {
+      .chi-tabs_item {
+        padding-left: 2.25rem;
+
+        .chi-icon {
+          font-size: $text;
+          height: 0.875rem;
+          left: 1rem;
+          width: 0.875rem;
+        }
+      }
+
+      .chi-tabs__subtabs {
+        > li {
+          > a {
+            padding-left: 3.25rem;
+            position: relative;
+
+            &::before {
+              border-bottom: 0.0625rem dotted set-color(grey, 100);
+              border-left: 0.0625rem dotted set-color(grey, 100);
+              content: '';
+              height: 0.6875rem;
+              left: 2.375rem;
+              position: absolute;
+              top: 0.125rem;
+              width: 0.6875rem;
+              z-index: 1;
+            }
+          }
+
+          .chi-tabs__subtabs {
+            > li {
+              > a {
+                padding-left: 4.25rem;
+                position: relative;
+
+                &::before {
+                  left: 3.375rem;
+                }
+              }
+            }
+          }
+        }
+      }
+
+      &.-active {
+        .chi-tabs_item {
+          .chi-icon {
+            left: 1rem;
+          }
+        }
+      }
+    }
+
+    &:not(.-sliding) > li.-active:not(.-has-active) > a {
+      background-color: $portal-color-background-table-tr-selected;
+
+      &::after {
```

Comment:
Shouldn't this be rendered as an icon button?

Reply:
It is not clear enough in the design. I display it as an pseudo-element when the -active state is present.